contracts/test/vault/index.js

```diff
@@ -32,7 +32,19 @@ describe("Vault", function () {
     const oracleAddresses = await getOracleAddresses(hre.deployments);
     const origAssetCount = await vault.connect(governor).getAssetCount();
     expect(await vault.isSupportedAsset(ousd.address)).to.be.false;
-    await oracleRouter.setFeed(ousd.address, oracleAddresses.chainlink.DAI_USD);
+
+    const latestBlock = await hre.ethers.provider.getBlock();
+    /* Mock oracle feeds report 0 for updatedAt data point. Set
+     * maxStaleness to current block time + 1 day to make the Oracle
+     * feeds valid
+     */
+    const maxStaleness = latestBlock.timestamp + 24 * 60 * 60;
```

Comment:
This is confusing because `maxStaleness` is duration of time, not a point in time. Maybe just have it be a couple years?

Reply:
Agreed